frontend/src/data/reducer.js

```diff
@@ -10,6 +11,7 @@ const reducer = (state = initialState, action) => {
     case types.SUCCESS:
       return {
         id: state.id || payload.id,
+        imageURL: state.imageURL || payload.imageURL,
```

Comment:
I think it's okay to just do imageURL: payload.imageURL because if it's not successfully, it will go back to the original state. Same thing with the id part

Reply:
The  reason I did it like this is that there  are now two state updates (id and imageURL). If one was already set, then I don't want it to be overridden with undefined in the other state update. Does that seem right?

Reply:
Ah I see. I think it will still keep the previous updated state cuz it depends on what you were updating. I haven't tried it tho